resources/ory/profile-evaluation.yaml

```
@@ -5,24 +5,32 @@ global:
       persistence:
         enabled: false
         postgresql:
-          enabled: false
+          enabled: true
         gcloud:
           enabled: false
 hydra:
   replicaCount: 1
   hydra:
-    autoMigrate: false
+    autoMigrate: true
```

This is also part of the ory azure-lite override.
```suggestion
    autoMigrate: false
```